### ISSUE TITLE
import fails on Manjaro Linux Gnome

### ISSUE THREAD

**Issue by mcsimenc** [ImageMagick/ImageMagick#2909]
### Prerequisites

- [ x] I have written a descriptive issue title
- [ x] I have verified that I am using the latest version of ImageMagick
- [ x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description / Steps to Reproduce
Hi, when I run the `import` command it fails, reporting:
```
import:  `import' @ error/import.c/ImportImageCommand/1285.
```

with no other information. What could be the problem?

Thanks

Matt

### System Configuration

- ImageMagick version: 7.0.10-45
- Environment (Operating system, version and so on): Manjaro Linux 20.2 running Gnome
- Additional information:

**Comment by urban-warrior** [comment]
You did not specify your import command-line.  We'll need that before we can comment further.

**Comment by mcsimenc** [comment]
Sorry...I assumed simply running `import` would display a help. When I use `import test.png` it runs. However, the crosshairs don't appear over the entire screen, only on certain windows like my web browser. Should I open a new issue?

**Comment by urban-warrior** [comment]
We have no control over the crosshairs cursor.  We request a cursor in the API and rendering the cursor is in the hands of the OS or hardware.

**Comment by mcsimenc** [comment]
Ok, I see. It's not just the crosshairs cursor, though, I'm unable to capture any part of the screen where the crosshairs cursor is not displayed. Is that also an OS/hardware issue?

**Comment by urban-warrior** [comment]
You can click and drag the cursor to capture portions of the screen or you can use the -screen option to capture the entire screen.

**Comment by mcsimenc** [comment]
It doesn't respond to click and drag unless over a window where the crosshairs appear. For example it still allows me to click on the launcher icons which behave as usual. Also several times now my computer has completely frozen when running `import test.png` and clicking on the part of the screen without crosshairs.

**Comment by mcsimenc** [comment]
And if I click and drag on the portion of the screen with the crosshairs I get this error:
```
import: unable to read X window image '': Resource temporarily unavailable @ error/xwindow.c/XImportImage/5041.
import:  `test.png' @ error/import.c/ImportImageCommand/1285.
```

**Comment by urban-warrior** [comment]
Try this command: `import -window root test.png`

**Comment by mcsimenc** [comment]
I get the same error:
```
$ import -window root test.png
import: unable to read X window image 'root': Resource temporarily unavailable @ error/xwindow.c/XImportImage/5040.
import:  `test.png' @ error/import.c/ImportImageCommand/1285.
```

**Comment by mcsimenc** [comment]
Sorry, that error was from using imagemagick v. 7.0.10.43. Here is the error from v7.0.10.45
```
$ ./magick import -window root test.png
import: unable to read X window image 'root': Resource temporarily unavailable @ error/xwindow.c/XImportImage/5041.
import:  `test.png' @ error/import.c/ImportImageCommand/1285.
```

**Comment by urban-warrior** [comment]
Unfortunately we cannot help further.  Its possible that your window system blocks screen capture through the X Windows system.    We're on a CentOS system and screen capture with `magick import` works flawlessly.

**Comment by mcsimenc** [comment]
Ok, thanks for the help!
